cc_licenses/templates/includes/languages_dropdown.html

```diff
@@ -0,0 +1,40 @@
+{% load bidi i18n %}
+{% get_current_language as LANGUAGE_CODE %}
+
+<div class="locale level-right level-item has-text-black" >
+  {% trans "Languages available" %}
+  <div class="control margin-{% start %}-small">
+    <div class="select">
+      <select id="languages-dropdown">
+        {% block lang-options %}
+        <option disabled>{% trans "Select" %}</option>
+
+        {% for option in languages_and_links %}
+          <option
```

Comment:
If `Traditional Chinese` is selected the label for the locale dropdown does not show `Traditional Chinese`. Instead the label shows `Arabic` (but the page does change to Traditional Chinese.

Reply:
Wow, good catch! I think it should be fixed now.